polynote-frontend/polynote/ui/component/notebook/cell.ts

```diff
@@ -139,24 +147,22 @@ abstract class Cell extends Disposable {
     protected id: number;
     protected cellId: string;
     public el: TagElement<"div">;
+    protected readonly cellState: StateHandler<CellState>;
 
-    protected constructor(protected dispatcher: NotebookMessageDispatcher, protected notebookState: NotebookStateHandler, protected cellState: StateHandler<CellState>) {
+    protected constructor(protected dispatcher: NotebookMessageDispatcher, protected notebookState: NotebookStateHandler, cell: StateHandler<CellState>) {
         super()
+        const cellState = this.cellState = cell.fork(this);
```

Comment:
any reason to keep `const cellState` around?

Reply:
It's used locally. Trying to minimize the merge pain of this change with all the other PRs that are waiting.